genie-web/src/main/java/com/netflix/genie/web/data/repositories/jpa/JpaCommandRepository.java

```diff
@@ -57,7 +57,8 @@
             + "SELECT DISTINCT(command_id)"
             + " FROM jobs"
             + " WHERE command_id IS NOT NULL"
-            + ");";
+            + ")"
+            + " AND id NOT IN (SELECT DISTINCT(command_id) FROM clusters_commands);";
```

Comment:
Indentation is different from similar statement a couple of lines above.

Reply:
yeah it's cause of if you flattened it out the spaces need to be there for the SQL to be valid. I could put extra space between `NULL` and `)` but figured this was the "best" I could do